internal/app/connectconformance/client_runner.go

```diff
@@ -32,8 +32,11 @@ const (
 	maxClientResponseSize = 16 * 1024 * 1024 // 16 MB
 )
 
-var errClosed = errors.New("send-to-client is closed")
-var errDuplicate = errors.New("duplicate test case")
+var (
+	errClosed    = errors.New("could not send request: client stdin is closed")
+	errDuplicate = errors.New("duplicate test case")
+	errNoOutcome = errors.New("no outcome ever received")
```

Comment:
This is just for consistency with errors. We had two places trying to indicate the same thing, but using different error wording. So now I've factored it into an error var that is used from both places.